### ISSUE TITLE
SplitterDistance must be between Panel1MinSize and Width - Panel2MinSize.

### ISSUE THREAD

**Issue by TWDI** [gitextensions/gitextensions#11085]
### Environment

- Git Extensions 4.1.0.16698
- Build 1fe52a137acef1fbd2b653dea263b44ad5844177
- Git 2.41.0.windows.1
- Microsoft Windows NT 10.0.19045.0
- .NET 6.0.18
- DPI 120dpi (125% scaling)
- Portable: False
- Microsoft.WindowsDesktop.App Versions

```
    Microsoft.WindowsDesktop.App 3.1.18 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 3.1.22 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 3.1.23 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 3.1.32 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 5.0.17 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 6.0.18 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

### Issue description

StackTrace (taken from later opened issue #11522 ):

```
System.InvalidOperationException: SplitterDistance must be between Panel1MinSize and Width - Panel2MinSize.
   at System.Windows.Forms.SplitContainer.set_SplitterDistance(Int32 value)
   at GitUI.CommandsDialogs.FormBrowse.SetSplitterPositions() in C:\projects\gitextensions\GitUI\CommandsDialogs\FormBrowse.cs:line 2222
   at GitUI.CommandsDialogs.FormBrowse.OnLoad(EventArgs e) in C:\projects\gitextensions\GitUI\CommandsDialogs\FormBrowse.cs:line 473
   at System.Windows.Forms.Form.OnCreateControl()
   at System.Windows.Forms.Control.CreateControl(Boolean fIgnoreVisible)
   at System.Windows.Forms.Control.CreateControl()
   at System.Windows.Forms.Control.WmShowWindow(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   at System.Windows.Forms.ContainerControl.WndProc(Message& m)
   at System.Windows.Forms.Form.WmShowWindow(Message& m)
   at System.Windows.Forms.Form.WndProc(Message& m)
   at ResourceManager.GitExtensionsFormBase.WndProc(Message& m) in C:\projects\gitextensions\ResourceManager\GitExtensionsFormBase.cs:line 106
   at GitUI.CommandsDialogs.FormBrowse.WndProc(Message& m) in C:\projects\gitextensions\GitUI\CommandsDialogs\FormBrowse.cs:line 565
   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, WM msg, IntPtr wparam, IntPtr lparam)
```

When I start Git Extentions, I get this error message.
I've fully deinstalled it, restarted and installed the same version again but it didn't help.

This seems like an issue issue which should be solved in an earlier version

### Steps to reproduce

Just start Git Extensions via it's shortcut generates this error

### Did this work in previous version of GitExtensions?

It worked fine with this version until yesterday.

**Comment by mstv** [comment]
Do the following steps resolve the issue?
- ignore the error message
- optionally: arrange the splitters
- close (which saves the current positions to the file `%APPDATA%\Roaming\GitExtensions\GitExtensions\GitExtensions.settings`)

**Comment by TWDI** [comment]
Hi Michael,

when I ignored the message, GITExt opened with an empty window. This
happened too (and was annoying) when I opened a submodule from our main
project.
Since I've toggled all the options the issue is gone:


[image: image.png]


So your info helped to solve the problem but there is a scenario where it
can go wrong.

Thanks for your help


Twan Dieltjes


Op vr 30 jun 2023 om 11:04 schreef Michael Seibt ***@***.***>:
